ccd-definition/ComplexTypes/CareSupervision/HearingOrders/HearingOrderForApproval.json

```diff
@@ -172,5 +181,14 @@
     "ElementLabel": "Hearing UUID",
     "SecurityClassification": "Public",
     "FieldShowCondition": "status=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingOrderForApproval",
+    "ListElementCode": "uploaderEmail",
+    "FieldType": "Text",
+    "ElementLabel": "Uploader email",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "status=\"DO_NOT_SHOW\""
   }
```

Comment:
These two additional fields should be added to `HearingOrderForReview.json` and `HearingOrder.json` as well.